enterprise/models.py

```diff
@@ -975,6 +975,30 @@ def get_program(self, program_uuid):
             return None
         return CourseCatalogApiServiceClient(self.enterprise_customer.site).get_program_by_uuid(program_uuid)
 
+    def get_course_run_enrollment_url(self, course_run_key):
```

Comment:
We need to modify the code [here](https://github.com/edx/edx-enterprise/blob/master/enterprise/api/v1/serializers.py#L217) to use these new methods.

Reply:
should we create a small subtask for that?

Reply:
@asadiqbal08 I'm fine with you creating a subtask if it helps you to get it done faster.

Reply:
created https://openedx.atlassian.net/browse/ENT-718